### PR TITLE
generalize conv2d, avg_pool2d, max_pool2d to support 1d and 3d cases

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

File: src/frontends/pytorch/src/op/avg_poolnd.cpp [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    auto pads = context.const_input<Shape>(3);  // pytorch supports only symmetric padding
+    auto rounding_type = context.const_input<bool>(4) ? ov::op::RoundingType::CEIL : ov::op::RoundingType::FLOOR;
+    auto count_include_pad = context.const_input<bool>(5);
+    FRONT_END_OP_CONVERSION_CHECK(context.input_is_none(6),
+                                  "Translation for aten::avg_pool2d do not support divisor_override input.");
+    // Although ov::AvgPool provides exclude_pad=false,
+    // The corner case of Average Pooling with ceil_mode on
+    // PyTorch allows sliding window go off bound, which leads to this accommodation.
+    // More detail on https://github.com/pytorch/pytorch/issues/57178
+    if (count_include_pad){
```
Comment (mvafin) [inline src/frontends/pytorch/src/op/avg_poolnd.cpp]: please fix code style

Comment (eaidova) [reply src/frontends/pytorch/src/op/avg_poolnd.cpp]: fixed

File: src/frontends/pytorch/src/op/max_poolnd.cpp [hunk 2]
```diff
@@ -11,19 +11,18 @@ namespace frontend {
 namespace pytorch {
 namespace op {
 
-OutputVector translate_max_pool2d(NodeContext& context) {
+OutputVector translate_max_poolnd(NodeContext& context) {
     auto kernel = context.const_input<Shape>(1);
     auto strides = context.const_input<Strides>(2);
-    auto pads_begin = context.const_input<Shape>(3);  // FIXME: The same 3 is used twice
-    auto pads_end = context.const_input<Shape>(3);    // FIXME: The same 3 is used twice
+    auto pads = context.const_input<Shape>(3); // pytorch supports only symmetric paddings
```
Comment (mvafin) [inline src/frontends/pytorch/src/op/max_poolnd.cpp]: Please fix codestyle

Comment (eaidova) [reply src/frontends/pytorch/src/op/max_poolnd.cpp]: fixed

**Review by mvafin — APPROVED** [review]

**Pull request merged**